clevis/Sanity/kernel-keyring-support/runtest.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+# vim: dict+=/usr/share/beakerlib/dictionary.vim cpt=.,w,b,u,t,i,k
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+#   runtest.sh of /CoreOS/clevis/Sanity/kernel-keyring-support
+#   Description: Test clevis to provide a parameter for the key description of a key in the kernel keyring
```

Comment:
Same comment as before regarding description

Reply:
Provided description according to your proposal.